### ISSUE TITLE
Launch OrgBook BC based on AVCR Technology

### ISSUE THREAD

**Issue by swcurran** [bcgov/von#285]
Tasks and activities related to launching the new OrgBook BC based on Indy Catalyst.

@WadeBarnes to lead the effort as a top priority for the team.

**Comment by WadeBarnes** [comment]
AVOB `prod` it up and connected to Sovrin Main Net.  Initialized wallet backup created.

AVIA `prod` is waiting on `aca-py v0.4.1` so we're not using images built off various branches.  Wallet has been imported from the BC-Reg-X-Agent and an initialized wallet backup has been created.

Agent updates will need to be deployed when available and before AVIA is fully brought online.

**Comment by WadeBarnes** [comment]
AVIA `prod` has been online for some time now, and was recently updated with the latest code; [6dd71ead125658f18a4e55862e983709fd359813](https://github.com/bcgov/von-bc-registries-agent/tree/6dd71ead125658f18a4e55862e983709fd359813)

AVOB `prod`:
- Rebuilding search indexes due to updates in index schema and data being indexed.
- Code is a few commits behind do to an issue with credential verification in the latest code.

**Comment by ianco** [comment]
Note we will need an updated release of aca-py (0.5.3) as the error with proof requests is fixed but an update to aca-py (completed and merged into master branch)

**Comment by WadeBarnes** [comment]
Indexes have been fully rebuilt and are in sync; https://orgbook-prod.pathfinder.gov.bc.ca/api/quickload
